servers.json

```diff
@@ -1065,5 +1065,12 @@
         "addresses": [
             "kingdomserver.be"
         ]
+    },
+     {
```

Comment:
Please remove the space before the bracket

Reply:
Should be fixed